build/Linux+BSD/mscore.1.in

```diff
@@ -339,6 +339,8 @@ internal file sanity check log (JSON)
 MPEG Layer III (lossy compressed audio)
 .It Li mpos
 measure positions (XML)
+.It Li mposx
```

Comment:
if you’re going to keep the old `.mpos` unchanged… maybe `.mpx` (and `.spx`) for the measure/segment pixel position ones?

Reply:
imho, according to previous discussion, those formats will not stay, as indeed, according to @dmitrio95 remarks, png  export is related to "DPI"[ppi] settings and that means dependent upon knowing the user config.